server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/MIMEMessageConverter.java

```diff
@@ -67,6 +67,7 @@
 import com.google.common.io.ByteStreams;
 import com.google.common.net.MediaType;
 
+@SuppressWarnings("deprecation")
```

Comment:
```
MessageBuilder messageBuilder = MessageBuilder.create();
```

Looks like it would need to use Message.Builder messageBuilder = Message.Builder.of();

:trollface: